Gopkg.toml

```diff
@@ -0,0 +1,38 @@
+
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#  name = "github.com/x/y"
+#  version = "2.4.0"
+
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/giantswarm/microerror"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/giantswarm/micrologger"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/giantswarm/vaultrole"
+
+[[constraint]]
+  name = "github.com/hashicorp/vault"
+  version = "0.7.3"
```

Comment:
The weird thing here is that I have to explicitly set the vault version even though `vaultrole` defines it already. If I do not set it here again the latest tag is used. Is this expected?

Reply:
Maybe vaultrole is still cached.

You can try `dep status` to do some debugging.

Reply:
Status is interesting. I am just not sure how it is computed. I tried to remove the Vault constraint here now because status looked fine but now after ensure and update the status shows 0.9.1. At the very beginning it had 0.9.3. This is weird. I will just stick to this now.